docs/contributing.md

```diff
@@ -58,7 +58,8 @@ commit. You can do that by creating a new commit on top of the initial commit
 `jj git push`
 will automatically force-push the bookmark.
 
-When your first PR has been approved, we typically give you contributor access,
+When your first PR has been approved, we invite you to the
```

Comment:
nit: Keep "typically"? That was supposed to indicate that it's not guaranteed that we give them contributor access. In particular, I usually don't do it for a simple documentation update.

Reply:
done. 

> In particular, I usually don't do it for a simple documentation update.

Yes, I also started to land doc updates from non-longterm contributors like you.